ui/src/main/java/edu/wpi/grip/ui/codegeneration/Language.java

```diff
@@ -0,0 +1,20 @@
+package edu.wpi.grip.ui.codegeneration;
+
+public enum Language {
+	JAVA("Java"), CPP("C++"), PYTHON("Python");
+	public final String name;
+	Language(String name){
+		this.name = name;
+	}
+	public static Language get(String name){
+		if(name.equals(JAVA.name))
+			return JAVA;
+		else if(name.equals(CPP.name))
+			return CPP;
+		else
+			return PYTHON;
```

Comment:
I do not think we should default to an actual language because it will make errors harder to find.